todo/src/Components/Todos.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { addTodo, toggleTodo, deleteTodo } from '../Actions'
+
+
+class Todos extends React.Component {
+    state = {
+        newTodo: ''
+    };
+
+    handleChanges = e => {
+        this.setState({ newTodo: e.target.value })
+    }
+    addTodo = e => {
+        e.preventDefault();
+        this.props.addTodo(this.state.newTodo);
+        this.setState({ newTodo: '' })
```

Comment:
It would also be helpful to check if your `newTodo` input is empty before running the `addTodo` action creator in order to avoid having a bunch of empty `h4` tags in your todos area (they won't be visible with your current styling, but it's good practice to account for this especially when you have more obvious styling for other implementations).